compiler/damlc/tests/daml-test-files/ExceptionSemantics.daml

```diff
@@ -3,7 +3,7 @@
 
 -- @SINCE-LF-FEATURE DAML_EXCEPTIONS
 -- @ERROR range=130:1-130:23; Unhandled exception: ExceptionSemantics:E
--- @ERROR range=145:1-145:25; Unhandled exception: ArithmeticError while evaluating (DIV_INT64 1 0).
+-- @ERROR range=145:1-145:25; ArithmeticError while evaluating (DIV_INT64 1 0).
```

Comment:
Can we keep the full error here to make sure this gets propagated as an `Unhandled exception`?